panther_driver/src/panther_can.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/python3
+
+import canopen
+from time import time, sleep
+from threading import Thread
+
+import rospy
+
+from constants import LEFT_WHEEL, RIGHT_WHEEL, VOLT_CHANNEL, AMP_CHANNEL
+
+
+class MotorController:
+    def __init__(self, can_node_id, eds_file) -> None:
+        self.wheel_pos = [0.0, 0.0]
+        self.wheel_vel = [0.0, 0.0]
+        self.wheel_curr = [0.0, 0.0]
+        self.battery_data = [0.0, 0.0] # V, I
+        self.runtime_stat_flag = [0, 0]
+        self.fault_flags = 0
```

Comment:
I'm not sure if those should be public members of the class. If this is only a container for few fields, then it's not really a necessary class

Reply:
:+1: